src/pudl/transform/ferc1.py

```diff
@@ -5849,12 +5877,28 @@ def add_dimension_total_calculations(
 ) -> pd.DataFrame:
     """Define dimension total calculations.
 
-    In addition to calculations defining how values reported in one set of facts can
-    be aggregated resulting in a value in another fact, there are implied calculation
-    relationships between values reported within a dimension. In particular, when a
-    dimension reports a ``total`` value, we assume it is the sum of all the non-total
-    values for that dimension. This function makes these within-dimension calculation
-    relationships explicit by defining new calculations for each dimension's ``total``.
+    If a factoid is marked as a total along some dimension, then that's
+    implicitly a calculation. Make these explicit.
+
+    The multi-dimension totals logic is a little hard to follow, so a concrete
```

Comment:
I'm happy to pair to walk through this logic - it makes more sense to *me* than the old version but that is no guarantee that it makes sense to anyone else...